raml/schemas/definitions/project.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "definitions":{
+        "project-input":{
+            "type": "object",
+            "properties": {
+                "public": {"$ref": "../definitions/container.json#/definitions/public"},
+                "label": {"$ref": "../definitions/container.json#/definitions/label"},
+                "metadata": {"$ref": "../definitions/container.json#/definitions/metadata"},
+                "group":{"$ref":"../definitions/group.json#/definitions/name"},
```

Comment:
This will be the group's `_id`, not it's `name`. (`scitran` vs `Scientific Transparency`)